.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -1,9 +1,11 @@
-name: "💡 Feature Request"
-description: Create a ticket requesting a new feature
-title: "💡 [PLUGIN] - <title>"
+name: Feature Request
+description: Suggest an idea for this project
+title: "[name of the plugin e.g. kube-monitoring] Issue Title"
```

```suggestion
title: "[name of the plugin] Issue Title"
```

would not give examples here, examples are the plugin names, which are in already